_source/logzio_collections/_log-sources/cloudwatch.md

```diff
@@ -511,6 +506,45 @@ To destroy the resources, use the following:
 ```bash
 terraform destroy
 ```
+</div>
+
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="test-events">
+
+#### Working with test events
+
+You can generate test events using the Logz.io Lambda test events generator and add these events to your Lambda function. This functionality is currently only available on Linux & macOS.
+
+
+<div class="tasklist">
+
+##### Generate a test event
+
+In your terminal, run the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/logzio/logzio_aws_serverless/tree/master/python3/cloudwatch/test_events/test_event_generator.sh)
+```
```

Comment:
In my computer it looks like this:
![image](https://user-images.githubusercontent.com/37478344/152013199-916b0a07-e5a5-4fc1-9a28-16916687aaf4.png)
Any idea how we can make it look better?